src/applications/login/components/SignInButtons.jsx

```diff
@@ -11,67 +11,92 @@ function loginHandler(loginType) {
   login(loginType, 'v1');
 }
 
-export default function SignInButtons({ isDisabled }) {
+export default function SignInButtons({ isDisabled, loginGovEnabled }) {
   return (
     <div>
+      {loginGovEnabled && (
+        <button
+          disabled={isDisabled}
+          type="link"
```

Comment:
Should these be `type="button"`?

Reply:
According to the W3C it should be a link https://w3c.github.io/aria/#link.

_If pressing the link triggers an action but does not change browser focus or page location, authors are advised to consider using the `button` role instead of the `link` role_

Reply:
@AGarcia-Clarity `link` isn't a valid type attribute, I think you're looking for `role="link"`, however, a link requires an `href` value, which we don't have here so to make this valid it either needs to be a normal button or an actual `<a>` element